src/app/wallets/update-pending-invoices.ts

```diff
@@ -190,12 +190,14 @@ const updatePendingInvoice = async ({
         sats: roundedDownReceived,
         displayCurrencyPerSat,
       })
-    } else {
+      return true
+    }
+
+    if (cents) {
```

Comment:
Doesn't this change open up the possibility for now `notificationService` call to happen?

If this change was just for the lint, I'd probably leave as-is for now since @UncleSamtoshi should be cleaning this up in #1241 I believe

Reply:
No, it does not open that possibility. Also using ! is worst because if that case exists then it will crash the app. The interface and implementation are specifying that the value is required.